src/main/scala/uk/gov/hmrc/SbtSassCompiler.scala

```diff
@@ -120,7 +133,13 @@ object SbtSassCompiler extends AutoPlugin {
             (anySassFileChanged: Boolean, _) =>
               if (anySassFileChanged || (sassFilesFound.nonEmpty && existingCssFromTargetDirectory.isEmpty)) {
                 logger.info("sbt-sass-compiler: Uncompiled Sass files found")
-                compileSassToCssFiles()
+                trySassCompilerFactoryBundled match {
+                  case true => { compileSassToCssFiles() }
+                  case false => {
+                    sassCompilerFactoryBundled = SassCompilerFactory.bundled() // if bundled fails the first time round, then attempting a re-trigger of bundled
```

Comment:
here rather than retry assigning it, I think if we kept compileSassToCssFiles the same as it is now, but caught an exception from it after running and if it matches the error of missing sass executable then as well as rerunning we need to first remove the whole directory that dart-sass is getting unpacked too so that next run should work